sensor/common/delegatedregistry/delegated_registry_handler.go

```diff
@@ -158,3 +160,22 @@ func (d *delegatedRegistryImpl) SetCentralGRPCClient(cc grpc.ClientConnInterface
 	d.imageSvc = v1.NewImageServiceClient(cc)
 	log.Debugf("Received central GRPC client connection")
 }
+
+func (d *delegatedRegistryImpl) processImageIntegrations(iiReq *central.ImageIntegrations) error {
+	select {
+	case <-d.stopSig.Done():
+		return errors.New("could not process updated image integrations, stop requested")
+	default:
+		log.Infof("Received %d updated and %d deleted image integrations", len(iiReq.GetUpdatedIntegrations()), len(iiReq.GetDeletedIntegrationIds()))
+
+		// Spawn a goroutine so that this handler doesn't block other messages from being processed
+		// while waiting for upserts to occur.
+		go d.doImageIntegrationUpdates(iiReq)
```

Comment:
realistically, how fast/slow will this be. If it's fast, then I would just process it synchronously to avoid running into potential race conditions from two image integration messages

Reply:
Initially was concerned about the 'wait for repo list' scenario. But since thats being removed, agreed, will remove this.